media-tv/kodi/kodi-17.0_alpha3.ebuild

```diff
@@ -0,0 +1,267 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI="6"
```

Comment:
All of the stuff from the previous ebuild also applies here, I won't repeat it again